src/rules.jl

```diff
@@ -112,18 +111,24 @@ _abs_deriv(x) = signbit(x) ? -one(x) : one(x)
 
 # unary #
 #-------#
-
-@define_diffrule SpecialFunctions.erf(x)         = :(  (2 / sqrt(π)) * exp(-$x * $x)       )
+@define_diffrule SpecialFunctions.erf(x) = :( 2 * exp(-$x * $x) * $(DiffRules.invsqrtπ) )
+@define_diffrule SpecialFunctions.erfinv(x) = :( (exp(SpecialFunctions.erfinv($x)^2) * $(DiffRules.sqrtπ)) / 2 )
+@define_diffrule SpecialFunctions.erfc(x) = :( -(exp(-$x * $x) * $(DiffRules.invsqrtπ)) * 2 )
+@define_diffrule SpecialFunctions.erfcinv(x) = :( -(exp(SpecialFunctions.erfcinv($x)^2) * $(DiffRules.sqrtπ)) / 2 )
+@define_diffrule SpecialFunctions.erfi(x) = :( 2 * exp($x * $x) * $(DiffRules.invsqrtπ) )
+@define_diffrule SpecialFunctions.erfcx(x) = :( 2 * ($x * SpecialFunctions.erfcx($x) - $(DiffRules.invsqrtπ)) )
+
+@define_diffrule SpecialFunctions.erf(x)         = :(  2 * (DiffRules.invsqrtπ * exp(-$x * $x))       )
```

Comment:
It's a bit inconsistent that you use `DiffRules....` instead of `$DiffRules` here.

Reply:
That's a typo :upside_down_face: